src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -687,6 +713,7 @@ public boolean canResume() {
     }
 
     @Override
+    @SuppressFBWarnings(value = "RC_REF_COMPARISON_BAD_PRACTICE_BOOLEAN", justification = "We want to explicitly check for boolean not-null and true")
```

Comment:
Is it wrong that I love that with `Boolean`, you effectively can treat it as a, I dunno, `troolean`? What'd be the term for a boolean but for three possible states... =)

Reply:
A trilean, acc. to Wikipedia.